gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
Seems the [Gradle 8.2](https://docs.gradle.org/8.2/release-notes.html) was just released a few days ago... We can test 8.2 in the following PR 👍